boskos/projects.json

```diff
@@ -0,0 +1,5 @@
+[
+{"name": "FAKE_PROJECT_1"},
```

Comment:
so that it can unmarshal into GCPProject struct, as I'm pretty sure I'll add more fields here.